packages/manager/apps/web/client/app/hosting/cdn/order/translations/Messages_fr_FR.json

```diff
@@ -1,12 +1,14 @@
 {
-  "hosting_cdn_order_title": "Commander un CDN",
-  "hosting_cdn_order_text": "L'option CDN active un réseau de serveurs de cache pour vos fichiers statiques (HTML, CSS, JavaScript). Il vous permet d'accélérer le chargement de vos pages web et d'utiliser les ressources de votre hébergement uniquement pour vos pages dynamiques.",
-  "hosting_cdn_order_duration": "Votre durée",
-  "hosting_cdn_order_duration_tooltip": "Regardless of where your new hosting package is set up, your domain name will continue to point to the current records. You will need to make the changes in your DNS zone when your new website is ready. This is an ideal choice if you do not want your current website to go down.",
-  "hosting_cdn_order_contracts": "Je reconnais avoir bien pris connaissance",
-  "hosting_cdn_order_contracts_of": "des",
-  "hosting_cdn_order_contracts_and": "et",
-  "hosting_cdn_order_submit_activate": "Activer",
-  "hosting_cdn_order_submit_pay": "Payer",
-  "hosting_cdn_order_cancel": "Annuler"
+  "hosting_cdn_order": {
+    "title": "Commander un CDN",
```

Comment:
:warning: Caution with this namepace feature because I'm not :100:% sure that our actual memsource configuration is compatible with. 

@blaryjp What's your opinion regarding our actual `.memsourcerc.yaml` configuration?

Thanks!

Reply:
You MUST NOT do that actually.

Reply:
I give it a try, I'll go back to the old way ;)

Reply:
@AxelPeter @blaryjp I think we should give a try as soon as the configuration can handle it properly. I will create a task for this.

Thanks again for this kind of enhancements/suggestions!